easybuild/easyconfigs/n/NCL/NCL-6.6.2-intel-2018b.eb

```diff
@@ -9,12 +9,19 @@ toolchainopts = {'cstd': 'c99', 'openmp': True, 'pic': True}
 
 source_urls = ['https://github.com/NCAR/ncl/archive/']
 sources = ['%(version)s.tar.gz']
-patches = ['NCL-6.4.0_fix-types.patch']
+patches = [
+    'NCL-6.4.0_fix-types.patch',
+]
 checksums = [
     'cad4ee47fbb744269146e64298f9efa206bc03e7b86671e9729d8986bb4bc30e',  # 6.6.2.tar.gz
     'f6dfaf95e5de9045745e122cb44f9c035f81fab92f5892991ddfe93945891c8f',  # NCL-6.4.0_fix-types.patch
 ]
 
+builddependencies = [
+    ('makedepend', '1.0.6'),
+    ('Bison', '3.0.5'),
```

Comment:
@casparvl `flex` should be included as a build dependency as well

Reply:
Should be done now...